libs/blocks/accordion/accordion.js

```diff
@@ -35,17 +76,26 @@ function createItem(accordion, id, heading, num) {
   }, heading.textContent);
   button.append(icon);
 
-  const panel = heading.nextElementSibling.firstElementChild;
-  const para = panel.querySelector('p');
-  const text = para ? para.textContent : panel.textContent;
+  const panel = heading.nextElementSibling?.firstElementChild;
 
+  const para = panel?.querySelector('p');
+  const text = para ? para.textContent : panel?.textContent;
```

Comment:
if `para.textContent` is empty do you want `panel.textContent`?

Reply:
i didn't actually write this part but i think they do?